From 9cded2f025f84978d455010d3ff708a762d9ef55 Mon Sep 17 00:00:00 2001
From: Julia Reynolds <juliacr@google.com>
Date: Wed, 31 Aug 2022 12:58:41 -0400
Subject: [PATCH] Lower per-app notificationchannel limit

Test: PreferencesHelperTest
Bug: 240422263
Change-Id: I8c12e3fc73e4a88842af275feaf2acffcced0402
(cherry picked from commit f528b337dd48b7e8071269e07e610bd4a3668c75)
(cherry picked from commit 39b4b1e9aad79c98413a866de53ccacccf64ca7f)
Merged-In: I8c12e3fc73e4a88842af275feaf2acffcced0402
---
 .../java/com/android/server/notification/PreferencesHelper.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/notification/PreferencesHelper.java b/services/core/java/com/android/server/notification/PreferencesHelper.java
index 477b8da61e0f..d8aa469bcd81 100644
--- a/services/core/java/com/android/server/notification/PreferencesHelper.java
+++ b/services/core/java/com/android/server/notification/PreferencesHelper.java
@@ -106,7 +106,7 @@ public class PreferencesHelper implements RankingConfig {
     private static final String NON_BLOCKABLE_CHANNEL_DELIM = ":";
 
     @VisibleForTesting
-    static final int NOTIFICATION_CHANNEL_COUNT_LIMIT = 50000;
+    static final int NOTIFICATION_CHANNEL_COUNT_LIMIT = 5000;
     @VisibleForTesting
     static final int NOTIFICATION_CHANNEL_GROUP_COUNT_LIMIT = 50000;
 
-- 
2.38.1.273.g43a17bfeac-goog

